src/tiled/command.cpp

```diff
@@ -50,6 +51,34 @@ QString Command::finalCommand() const
     return finalCommand;
 }
 
+QString Command::finalWorkingDir() const
+{
+    QString finalWorkingDir;
```

Comment:
This can't be right, it looks like `finalWorkingDir` will always return an empty string.